docs/Makefile

```diff
@@ -46,23 +54,31 @@ docs:  $(patsubst %.md,%,$(wildcard *[157].md))
 	 -e 's/\\$$/  /g' $<  | \
 	$(GOMD2MAN) -in /dev/stdin -out  $@
 
-.PHONY: install
-install: docs
+.PHONY: install_1
+install_1:
 	install -d ${DESTDIR}/${MANDIR}/man1
 	install -m 0644 ramalama*.1 ${DESTDIR}/${MANDIR}/man1
 	install -m 0644 links/ramalama*.1 ${DESTDIR}/${MANDIR}/man1
+
+
+.PHONY: install_5
+install_5:
 	install -d ${DESTDIR}/${MANDIR}/man5
 	install -m 0644 ramalama*.5 ${DESTDIR}/${MANDIR}/man5
+
+.PHONY: install_7
+install_7:
 	install -d ${DESTDIR}/${MANDIR}/man7
 	install -m 0644 ramalama*.7 ${DESTDIR}/${MANDIR}/man7
 
+.PHONY: install
+install: docs install_1 install_5 install_7
 	install -d ${DESTDIR}/${DATADIR}/ramalama
 	install -m 0644 ramalama.conf ${DESTDIR}/${DATADIR}/ramalama
 
-
 .PHONY: install-tools
 install-tools:
-	go install github.com/cpuguy83/go-md2man@latest
+	$(GO) install github.com/cpuguy83/go-md2man@latest
```

Comment:
This change makes sense. I question all the other changes checkmate has suggested. Same for this PR:

https://github.com/containers/ramalama/pull/1277

Reply:
> This change makes sense. I question all the other changes checkmate has suggested. Same for this PR:
> 
> #1276

May I ask to at least keep the 'test' rule in global Makefile? I think it is more "standard" than "tests" ...

Reply:
What you seems good alias test and tests, no need to break workflows people are using